src/service_name.json

```diff
@@ -234,6 +234,11 @@
     "AzureServiceName": "Digital Twins",
     "URL": "https://azure.microsoft.com/services/digital-twins"
   },
+  {
+    "Command": "az durabletask",
+    "AzureServiceName": "Durable Task",
+    "URL": ""
```

Comment:
Could you provide the URL for your service?

Reply:
We don't have a learn.microsoft link since our service is still in preview.